src/errorHandler.ts

```diff
@@ -62,10 +63,27 @@ export function registerDisposables(): Disposable[] {
  */
 function updateStatusBar(message: string): void {
     statusBarItem.text = message;
+    statusBarItem.tooltip = prettierInformation.toString();
     statusBarItem.show();
 }
 
 /**
+ * 
+ * @param module the module used
+ * @param version the version of the module
+ * @param bundled is it bundled with the extension or not
+ */
+export function setUsedPrettier(
+    module: string,
+    version: string,
+    bundled: boolean
+) {
+    prettierInformation = `Module: ${module} | Version: ${version} | Bundled with extension: ${bundled
```

Comment:
I find that tooltip quite long...

Reply:
It's long, but it gives all the information you need right? We can also just show the version, and in the output channel add this huge line? But it can change for different files so I don't know ¯\_(ツ)_/¯

Reply:
`prettier@1.7.0` instead?

Reply:
I like that!